src/api/js-api.domainsElement.test.js

```diff
@@ -1,3 +1,5 @@
+import { jest } from "@jest/globals";
```

Comment:
Otherwise you would get:
```
FAIL  src/api/js-api.imo.test.js
  ● Test suite failed to run

    TypeError: jest.describe is not a function

      1 | import { localStorageMock } from "./local-storage-mock";
      2 |
    > 3 | jest.describe("window.importMapOverrides", () => {
        |      ^
      4 |   let localStorageBackup;
      5 |
      6 |   const defaultMap = {

      at Object.describe (src/api/js-api.imo.test.js:3:6)
```

Reply:
Sorry, you are right @ivan-navarro-75. I can remove `jest.` from describe. Not sure why I have it there...

Reply:
Fixed.